sdk/attestation/azure-security-attestation/inc/azure/attestation/attestation_administration_client.hpp

```diff
@@ -282,6 +486,21 @@ namespace Azure { namespace Security { namespace Attestation {
         std::unique_ptr<Azure::Core::Http::RawResponse> const& serverResponse,
         AttestationTokenValidationOptions const& tokenValidationOptions) const;
 
+    Azure::Core::Http::Request CreateSetPolicyRequest(
+        Models::AttestationType const& attestationType,
+        Azure::Core::IO::BodyStream& bodyStream) const;
+
+    Response<Models::AttestationToken<Models::PolicyResult>> ProcessSetPolicyResponse(
+        AttestationTokenValidationOptions const& tokenOptions,
+        std::unique_ptr<Azure::Core::Http::RawResponse>& rawResponse) const;
+
+    Azure::Core::Http::Request CreateResetPolicyRequest(
+        Models::AttestationType const& attestationType,
+        Azure::Core::IO::BodyStream& bodyStream) const;
+    Response<Models::AttestationToken<Models::PolicyResult>> ProcessResetPolicyResponse(
+        AttestationTokenValidationOptions const& tokenOptions,
+        std::unique_ptr<Azure::Core::Http::RawResponse>& rawResponse) const;
```

Comment:
are these public?  It is a little hard to know from the PR.
If yes, do we need docs?

Reply:
If this was anything other than a proof of concept prototype, then they absolutely should be documented.

But this is a proof of concept prototype.